__tests__/app.test.js

```diff
@@ -137,3 +137,64 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+describe("PATCH api/reviews/:review", () => {
+  test("returns review object with updated votes", () => {
+    const input = {
+      inc_votes: 10,
+    };
+    return request(app)
+      .patch("/api/reviews/2")
+      .send(input)
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review.votes).toBe(15);
+        expect(review).toEqual(
+          expect.objectContaining({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+          })
+        );
+      });
+  });
+  test("status:404, responds with an error message when passed a object with wrong propreties", () => {
```

Comment:
Can you just amend these test descriptions please to state 400 instead of 404? 🙂